pkgs/applications/networking/cluster/terraform-providers/default.nix

```
@@ -14,6 +14,9 @@ let
       version = data.version;
       subPackages = [ "." ];
       doCheck = false;
+      # https://github.com/hashicorp/terraform-provider-scaffolding/blob/main/.goreleaser.yml
```

Pin URL (press <kbd>Y</kbd> on that Github page to get a stable link), write explicit motivation in addition to the link, like:

```suggestion
      # Upstream says:
      # > goreleaser does not work with CGO
      
      # from: https://github.com/hashicorp/terraform-provider-scaffolding/blob/a8ac8375a7082befe55b71c8cbb048493dd220c2/.goreleaser.yml
```